model-optimizer/mo/back/ie_ir_ver_2/emitter.py

```diff
@@ -12,6 +13,8 @@
 from mo.utils.utils import refer_to_faq_msg
 from mo.utils.version import get_version
 
+ET_defused = defuse_stdlib()[ET]
```

Comment:
still no comment. Please add it

Reply:
can we insert it into import block and do needed imports like:
'from ET_defused import Element, ...' so there will be no need to change the code below

Reply:
We cant import from defuse_stdlib, but we can assign everything in the beginning of file.